classes/Event/class.xoctEventGUI.php

```diff
@@ -1427,30 +1436,41 @@ protected function createHyperlinks(string $intro_text): string
         return $intro_text;
     }
 
-    protected function addCurrentUserToProducers(): void
+    /**
+     * Adds the current user to the specified group.
+     * Producers group is the default selected group. (PluginConfig::F_GROUP_PRODUCERS)
+     *
+     * @param string $group_config_name the group config name (default PluginConfig::F_GROUP_PRODUCERS)
+     *
+     */
+    protected function addCurrentUserToGroup(string $group_config_name = PluginConfig::F_GROUP_PRODUCERS): void
```

Comment:
I'd suggest to not make a default value `PluginConfig::F_GROUP_PRODUCERS` here. If we ever change that, one must check every location this method is used again. Or is there some case I'm missing where this default is needed?

Reply:
The function was initially only for the producers group, and it was built on producers group, therefore I have decided to make it that way, in order to minimize the changes and make it reusable.
I can explain it in a VC if you will!